subworkflows/local/prepare_genome.nf

```diff
@@ -43,6 +44,10 @@ workflow PREPARE_GENOME {
             BWAMEM2_INDEX(fasta)
             ch_bwa = BWAMEM2_INDEX.out.index
             ch_versions = ch_versions.mix(BWAMEM2_INDEX.out.versions)
+        } else if (params.aligner == "dragmap") {
+            DRAGMAP_HASHTABLE(fasta)
+            ch_bwa = DRAGMAP_HASHTABLE.out.hashmap
+            ch_versions = ch_versions.mix(DRAGMAP_HASHTABLE.out.versions)
         }
```

Comment:
Should we have a placeholder in igenomes for this hashtable?

Reply:
Even, I think it should be computed even when a bwa params is present

Reply:
so maybe change `bwa` param to mapping_reference or something?